modules/osd-aws-privatelink-firewall-prerequisites.adoc

```diff
@@ -0,0 +1,124 @@
+|Required
+
+|`quay.io`
+|Used by the cluster to download the platform container images
+|Required
+
+|`sso.redhat.com`
+|The `https://cloud.redhat.com/openshift` site uses authentication from `sso.redhat.com` to  download the pull secret and use Red Hat SaaS solutions to facilitate monitoring of your subscriptions, cluster inventory, chargeback reporting, and so on.
+|Required
+
+|`pull.q1w2.quay.rhcloud.com`
+|Provides a fallback registry used by the cluster when quay.io is not available.
+|Recommended
+|===
++
+When you add a site such as `quay.io` to your allowlist, do not add a wildcard entry such as `*.quay.io` to your denylist. In most cases, image registries use a content delivery network (CDN) to serve images. If a firewall blocks access, then image downloads are denied when the initial download request is redirected to a host name such as `cdn01.quay.io`.
+
+. Managed clusters require Telemetry to be enabled to allow Red Hat to react more quickly to problems and better support our customers, as well as better understand how product upgrades impact clusters.  
+See link:https://docs.openshift.com/container-platform/4.6/support/remote_health_monitoring/about-remote-health-monitoring.html[About remote health monitoring] for more information about how remote health monitoring data is used by Red Hat.
++
+[cols="2,4,2",options="header"]
+|===
+|URL | Function | Requirement
+
+|`https://cloud.redhat.com`
+|Used by the cluster for the `insights operator` that integrates with the SaaS Red Hat Insights
+|Required
+|===
+
+. For Amazon Web Services (AWS), you must grant access to the URLs that provide the AWS API and DNS:
++
+[cols="2,4,2",options="header"]
+|===
+|URL | Function | Requirement
+
+|`*.amazonaws.com`
+|Required to access AWS services and resources. Review the link:https://docs.aws.amazon.com/general/latest/gr/rande.html[AWS Service Endpoints] in the AWS documentation to determine the exact endpoints to allow for the regions that you use.
+|Required
+|===
+
+. Allowlist the following URLs:
++
+[cols="2,4, 2",options="header"]
+|===
+|URL | Function | Requirement
+
+|`mirror.openshift.com`
+|Required to access mirrored installation content and images. This site is also a source of release image signatures, although the Cluster Version Operator needs only a single functioning source.
+|Required
+
+|`storage.googleapis.com/openshift-release`
+|Alternative site to download platform release signatures that are used by the cluster to know what images to pull from quay.io.
+|Recommended
+
+|`*.apps.<cluster_name>.<shard>.<env>.openshiftapps.com`
+|When allowlisting domains, use this in your corporate network to reach applications deployed in OpenShift, or to access the OpenShift console. The <shard> and <env> values are known after install time, and  <cluster_name> is set by the customer.
+|Required
+
+|`*.api.<cluster_name>.<shard>.<env>.openshiftapps.com:`
+|Used to access the public endpoint for the cluster API server. 
+|Required
```

Comment:
```suggestion
```

@dustman9000 @blrm Can either of you validate the removal of these 2 URLs? According to @jharrington22, they _should_ be able to be removed

Reply:
Good catch, yes this can be removed as this is not a public endpoint.

Reply:
Thanks Dustin! @kmccarron-rh FYI ^^